### PR TITLE
0.15.2 release notes, manpage and version bump

### PR DESCRIPTION
**Pull request by Sjors**
Semi manual based on [commits since v0.15.1](https://github.com/bitcoin/bitcoin/compare/v0.15.1...0.15), so please sanity check.

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -62,15 +62,53 @@ frequently tested on them.
...
+ 
+### Wallet
+- #11647 `7af2457` contrib/init: Update openrc-run filename (Luke Dashjr)
+- #11647 `3f1db56` Wrap dumpwallet warning and note scripts aren't dumped (MeshCollider)
+- #11647 `42ea47d` Add wallet backup text to import*, add* and dumpwallet RPCs (MeshCollider)
+- #11647 `3a6cdd4` Add test for multiwallet batch RPC calls (Russell Yanofsky)
+- #11647 `1c8c7f8` Add missing batch rpc calls to python coverage logs (Russell Yanofsky)
+- #11647 `1036c43` Add missing multiwallet rpc calls to python coverage logs (Russell Yanofsky)
+- #11647 `2eea279` Make AuthServiceProxy._batch method usable (Russell Yanofsky)
+- #11647 `305f768` Limit AuthServiceProxyWrapper.__getattr__ wrapping (Russell Yanofsky)
```
Comment (hebasto) [inline doc/release-notes.md]: Do you mean:
```
- #11647 `2eea279` Make AuthServiceProxy.\_batch method usable (Russell Yanofsky)
- #11647 `305f768` Limit AuthServiceProxyWrapper.\_\_getattr\_\_ wrapping (Russell Yanofsky)
```

**Review by hebasto — CHANGES_REQUESTED** [review]

File: doc/release-notes.md (hunk 1 above)
Comment (Sjors) [reply doc/release-notes.md]: Thanks, added escape characters.

**Comment by Sjors** [comment]
Removed translation credits, because they weren't updated for this release.

**Comment by laanwj** [comment]
I think it'd make sense to combine this with #14263 and a bump-version commit

File: doc/release-notes.md [hunk 2]
```diff
@@ -62,15 +62,51 @@ frequently tested on them.
...
+- #11647 `1c8c7f8` Add missing batch rpc calls to python coverage logs (Russell Yanofsky)
+- #11647 `1036c43` Add missing multiwallet rpc calls to python coverage logs (Russell Yanofsky)
+- #11647 `2eea279` Make AuthServiceProxy.\_batch method usable (Russell Yanofsky)
+- #11647 `305f768` Limit AuthServiceProxyWrapper.\_\_getattr\_\_ wrapping (Russell Yanofsky)
+- #11647 `7026845` Fix uninitialized URI in batch RPC requests (Russell Yanofsky)
+- #11647 `6372a75` [Wallet] always show help-line of wallet encryption calls (Jonas Schnelli)
+
+### bitcoin-tx
+
+- #11662 `a69cc07` Sanity-check script sizes in bitcoin-tx
```
Comment (fanquake) [inline doc/release-notes.md]: #11662 is a backport of #11554 from TheBlueMatt

File: doc/release-notes.md [hunk 3]
```diff
@@ -62,15 +62,51 @@ frequently tested on them.
...
+the vulnerable versions to 0.15.2 or 0.16.3 as soon as possible.
+
+0.15.2 Change log
 =================
 
+### Build system
+
+- #13093 `1618c63` depends: Fix Qt build with XCode 9.2 and 9.3 (fanquake)
+- #13574 `9fd3e00` depends: Update Qt download url (fanquake)
+- #12032 `cb7ef31` Make boost::multi_index comparators const
```
Comment (fanquake) [inline doc/release-notes.md]: #12032 is a backport of #11847 from sdaftuar

**Comment by Sjors** [comment]
* updated pull request numbers to reference originals rather than the backport. This is especially useful for #11647 which was one backport PR multiple PR's
* added missing authors @fanquake pointed out
* added RPC and Tests category
* added a version update commit
* moved manpage commit from #14263

**Comment by Sjors** [comment]
From the manpage PR:
> I used the generator script from master and fixed the version number after running the script.
>
> Notable changes:
> * script verification threads changed from 6 to 4
> * maximum BIP141 block weight default changed from 3000000 to 3996000
> * blockmaxsize deprecated

**Comment by laanwj** [comment]
ACK 391b2cf0250de44afcaeaaf977e3ef8ec5c4d55f

Thank you for doing this!

**Pull request merged**